docs/develop/bundled-packages.md

```diff
@@ -11,3 +11,47 @@ python requirements. Note that the serving environment may itself be dependent
 on some standard Python modules such as `requests` and `kserve`, so it's best to
 avoid using package names that may conflict with any standard or popular Python
 packages.
+
+## External packages
+
+Where possible, a truss folder should be self contained. But, there are certain
+situations where you may need to keep some code outside. e.g. when you want
+to share code across multiple local trusses and publishing that code into a pypi
+package is not an option. External packages are meant for this use case, but
```

Comment:
why the hesitation to encourage its use? I actually think this should be more widely used than `bundled_packages`?